### ISSUE TITLE
Dynamically added handlers doesn't restores after killing by killer_hwm

### ISSUE THREAD

**Issue by kubrack** [erlang-lager/lager#428]
It's possible to dynamically configure handler and add it to sink via start_handler(Sink, Module, Config). But when the sink killed by killer_hwm it restores with startup configuration so dynamically added handler config lost. 
Since start_handler/3 is exported so is public API part, it would be consequentially to preserve it to config too, some like this:
 
```
--- a/src/lager_app.erl
+++ b/src/lager_app.erl
@@ -96,6 +96,8 @@ start_handlers(Sink, Handlers) ->
 start_handler(Sink, Module, Config) ->
     {ok, Watcher} = supervisor:start_child(lager_handler_watcher_sup,
                                            [Sink, Module, Config]),
+    AppConfHandlers = application:get_env(lager, handlers, []),
+    application:set_env(lager, handlers, [{Module, Config}|AppConfHandlers]),
     {Module, Watcher, Sink}.

```

**Comment by mrallen1** [comment]
We just merged a PR today which addresses this #459.  Would you please try it out? Thanks.

**Comment by mrallen1** [comment]
OK, closing this issue. Please open a new ticket if you need to.
